tools/SampleAnalyzer/Commons/Service/LogReport.h

```diff
@@ -84,7 +84,7 @@ class LogReport
  public:
 
   /// Constructor withtout argument
-  LogReport() : GeneralCounter_(0), GlobalThreshold_(1000000), MsgThreshold_(10)
+  LogReport() : GeneralCounter_(0), GlobalThreshold_(1), MsgThreshold_(1)
```

Comment:
@BenjaminFuks, these changes might be problematic for other portions of the software. It might be better to implement a silence raise option for the warnings where some of the reoccurring warning messages can be just presented in summary.

Reply:
I agree with the proposal.